src/libs/ReportUtils.js

```diff
@@ -476,7 +476,9 @@ function getIcons(report, personalDetails, policies, defaultIcon = null) {
         participantDetails.push([
             login,
             lodashGet(personalDetails, [login, 'firstName'], ''),
-            lodashGet(personalDetails, [login, 'avatar']) || getDefaultAvatar(login),
+            lodashGet(personalDetails, [login, 'avatarThumbnail']) ||
```

Comment:
Not actually sure if we want this - Not sure if we ever get other people's avatar thumbnails in Onyx?